ios/RNSScreenStackHeaderConfig.h

```diff
@@ -1,13 +1,35 @@
-#import <React/RCTConvert.h>
+#ifdef RN_FABRIC_ENABLED
+#import <React/RCTViewComponentView.h>
+#import <UIKit/UIKit.h>
+#import "RNSScreenStackHeaderSubviewComponentView.h"
+#else
 #import <React/RCTViewManager.h>
+// TODO: move this import when SearchBar is implemented on Fabric
+#import "RNSSearchBar.h"
+#endif
 
+#import <React/RCTConvert.h>
 #import "RNSScreen.h"
-#import "RNSSearchBar.h"
 
-@interface RNSScreenStackHeaderConfig : UIView
+@interface RNSScreenStackHeaderConfig :
+#ifdef RN_FABRIC_ENABLED
+    RCTViewComponentView
+#else
+    UIView
+#endif
 
 @property (nonatomic, weak) RNSScreenView *screenView;
 
+#ifdef RN_FABRIC_ENABLED
+// TODO: change type when StackHeaderSubview is merged (??)
+@property (nonatomic) NSMutableArray<RNSScreenStackHeaderSubviewComponentView *> *reactSubviews;
```

Comment:
Why is it needed if it is included here: https://github.com/software-mansion/react-native-screens/pull/1417/files#diff-e5ef5b6e29f17bca80b51bc0c5faef1a44bac24e00952b30ac822520213dc6a5R71

Reply:
Done here: [7cd80e9](https://github.com/software-mansion/react-native-screens/pull/1418/commits/7cd80e90050c029a900bbcb8e9651218e33b6f8f) (#1418) as it was mentioned there for the first time.

Reply:
I added comment in #1418 if there is a reason why it is in header and not in implementation. Could you answer it too?

Reply:
I resolve conversation here, as this prop can be completely merged only in #1418.
Will answer it there.